src/faction_camp.cpp

```diff
@@ -287,16 +287,16 @@ void talk_function::become_overseer( npc &p )
     p.companion_mission_role_id = "FACTION_CAMP";
     p.set_attitude( NPCATT_NULL );
     p.mission = NPC_MISSION_GUARD_ALLY;
-    p.chatbin.first_topic = "TALK_CAMP_OVERSEER";
+    p.chatbin.first_topic = "TALK_CAMP_GENERAL";
     p.set_destination();
-    get_basecamp( p );
+    get_basecamp( p.global_omt_location() );
 }
 
 
-void talk_function::camp_missions( mission_data &mission_key, npc &p )
+void basecamp::camp_missions( mission_data &mission_key, npc &p )
 {
     std::string entry;
-    basecamp *bcp = get_basecamp( p );
+    basecamp *bcp = get_basecamp( p.global_omt_location() );
```

Comment:
If we don't have a basecamp at this point, we can't be here, because camp_missions is now a basecamp member function.

Reply:
What else can i put besides that in its place to get the current basecamp?

Reply:
You should have the current basecamp before you call a basecamp function.  That's what get_basecamp is for.